data/themes/darktable.css

```diff
@@ -753,47 +755,61 @@ overshoot.right
 #main-histogram .toggle
 {
   border: 1.1px solid alpha(@button_bg, 0.3);
-  min-height: 1em;  /* FIXME: what happens if don't set these? -- what is the source */
   min-width: 1em;
 }
 
 #scope-type-button,
 #histogram-scale-button,
 #waveform-type-button
 {
-  color: alpha(@button_checked_fg, 0.56);           /* hack to match legacy color of white with alpha of 0.5 */
-  background-color: alpha(darker(@button_bg),0.8);  /* hack to be close to legacy color */
+  color: alpha(@button_checked_fg, 0.66);           /* hack to match legacy color of white of scope/histogram buttons */
+  background-color: alpha(darker(@button_bg), 0.75);  /* hack to be close to legacy color */
 }
 
 #scope-type-button > #button-canvas,
 #histogram-scale-button > #button-canvas,
 #waveform-type-button > #button-canvas
 {
-  /* hack to make icons larger to match legacy appearance */
+  /* set icons larger to match legacy appearance */
   margin: -14px; /* not real pixels, this is used as a percent extra space */
```

Comment:
This expands the buttons to be flush to the button edges. Do we want this? This means that some icons (linear/logarithmic) are slightly larger than before, so that the waveform icon also is flush to the edge.

Reply:
Indeed, we don't want/need this. I don't really see a difference with or without this line. Difference is tiny on my side. So I remove that too.